### DISCUSSION TITLE
Excerpts with Eleventy

### DISCUSSION THREAD

**Discussion by giscus** [jonathanyeong/personal-website#19] [Blog Post Discussions]
# Excerpts with Eleventy

A tutorial on getting excerpts into 11ty.

https://jonathanyeong.com/excerpts-with-eleventy/

**Comment by caraya** [comment]
Does this work with the current version of Eleventy? I've tried it following the instructions in your post but can't get it to work

**Reply by jonathanyeong** [reply]
Please share your project or the error you're getting. This post could definitely be outdated!

**Reply by caraya** [reply]
I went with another strategy to create the excerpt
